docs/userGuide/deployingTheSite.md

```diff
@@ -23,17 +23,28 @@
 
 1. Set the [`baseUrl` property of the `site.json` file](siteJsonFile.html#baseurl) to match the deploy location.
 1. (Optional) Use the [`markbind serve` command](cliCommands.html#serve-command) to stage the site locally and confirm the contents are as expected.
-1. Use the [`markbind build` command](cliCommands.html#build-command) to generate the site from source files. That command puts the generated site files in a directory named `_site` (you can change the output directory using parameters supplied to the command).
+1. (Optional) Use the [`markbind build` command](cliCommands.html#build-command) to generate the site from source files. That command puts the generated site files in a directory named `_site` (you can change the output directory using parameters supplied to the command). Note that the [`markbind deploy` command](cliCommands.html#deploy-command) generates the site from source files by default as well.
```

Comment:
same here.

Reply:
Fixed in [a27bad9](https://github.com/MarkBind/markbind/pull/2218/commits/a27bad98b61f596abe72f9dfc46d580724ddd2cf)!